apps/wallet-dashboard/lib/utils/migration/groupStardustObjectsByMigrationStatus.ts

```diff
@@ -58,25 +61,35 @@ interface SummarizeMigrationObjectParams {
     basicOutputs: IotaObjectData[] | undefined;
     nftOutputs: IotaObjectData[] | undefined;
     address: string;
+    resolvedObjects?: ResolvedObjectTypes[];
 }
 
 export function summarizeMigratableObjectValues({
```

Comment:
I bet this issue also happens in the unmigratable objects too, so we should replicate the same logic there too 🙏🏼